src/main/java/pages/ContactUsPage.java

```diff
@@ -62,13 +62,13 @@ public ContactUsPage validateWriteToUs(String userName, String phoneNumber, Stri
         MobileExplicitWaitFactories.click(writeToUs, WaitStrategy.CLICKABLE, "user clicked on write to us");
         MobileExplicitWaitFactories.click(name, WaitStrategy.CLICKABLE, "user click on name section");
         MobileExplicitWaitFactories.sendKeys(name, userName, WaitStrategy.VISIBLE, " their name");
-        MobileDriverManager.getDriver().navigate().back();
+        //MobileDriverManager.getDriver().navigate().back();
```

Comment:
we can remove the comments if it's not required

Reply:
ok

Reply:
In the case of an array, there may not be much clarity about which variant we are clicking in the test method since we use index of the array